### DISCUSSION TITLE
Exclude translations from publish

### DISCUSSION THREAD

**Discussion by Quadrubo** [Laravel-Lang/common#182] [Q&A] [answered]
My paginator uses it's own icons.
Therefor I removed the buildin icons from the translation file:

```php
<?php

declare(strict_types=1);

return [
    'next' => 'Weiter',
    'previous' => 'Zurück',
];
```

However whenever I run `lang:update` the icons come back:

```php
<?php

declare(strict_types=1);

return [
    'next' => 'Weiter &raquo;',
    'previous' => '&laquo; Zurück',
];
```

Can I exclude the keys from the update command?

**Comment by andrey-helldar** [comment, marked as answer]
Unfortunately, the publisher does not support excluding some phrases from processing. We gave up this practice a long time ago because it did not allow us to correctly maintain the actual state of translations in the final application.

Since this view for values is default, we are adding it "as is".

But you can, on the application side, modify it using the following call instead of the default one:

```php
{{ str_replace(‘&raquo;’, ‘’, __(‘pagination.next’)) }}

{{ str_replace(‘&laquo;’, ‘’, __(‘pagination.previous’)) }}
```

**Reply by Quadrubo** [reply]
Okay thanks for your answer. What do you think about excluding the strings &raquo; / &laquo; from these translations all together as (imo) they aren't translations and can be added by the user if needed?

**Reply by andrey-helldar** [reply]
We have removed special characters from the translation keys. You may still have some old keys. You can reset the files using the `php artisan lang:reset` console command.
